scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -37,74 +38,8 @@ class ThemeAggregator {
         return ThemeAggregator.writeJsFile(assets);
       }
     }
-    return null;
```

Comment:
I found a bug with the aggregate function when trying to default the theme in webpack. It wouldn't generate the theme because I didn't have a theme configuration && the reason I wanted to default the theme was so consumer did not need one 😅  I think the following needs changed:

```diff
static aggregate(theme) {
-   let themeConfig;
+  let themeConfig = {};
    const defaultConfig = path.resolve(process.cwd(), CONFIG);
    if (fs.existsSync(defaultConfig)) {
      // eslint-disable-next-line global-require, import/no-dynamic-require
      themeConfig = require(defaultConfig);
-      const assets = ThemeAggregator.aggregateThemes({ ...themeConfig, ...theme && { theme } });
-      if (assets) {
-        return ThemeAggregator.writeJsFile(assets);
-      }
    }
+   const assets = ThemeAggregator.aggregateThemes({ ...themeConfig, ...theme && { theme } });
+   if (assets) {
+     return ThemeAggregator.writeJsFile(assets);
+   }
    return null;
```

Reply:
hey, the doc explicitly says you need a terra-theme config! hahaa jk, i will update this

Reply:
https://github.com/cerner/terra-toolkit/pull/376/commits/1d1bb30aefe21cc8976ec3b164fd57947e4299bb#diff-1672fd55492cdce050f5aa314b17029bR31

Reply:
THANKS 😄